src/bookshelf/mapper.ts

```diff
@@ -28,6 +28,43 @@ export default class Bookshelf implements I.Mapper {
   }
 
   /**
+   * Recursively add all the (nested) relations to the template
+   * @param data
+   * @param type
+   * @param bookshelfOptions
+   * @param template
+   */
+  mapRelations(model: any, type: string, bookshelfOptions: I.BookshelfOptions = {relations: true}, template?: ISerializerOptions): any {
```

Comment:
Maybe the type of the function could be have `model` be of type `Model` and return `void`, since there's no return statement with a value.

```
mapRelations(model: Model, type: string, bookshelfOptions: I.BookshelfOptions = {relations: true}, template?: ISerializerOptions): void {
```